src/libraries/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -15,6 +16,11 @@ public class ProcessStandardConsoleTests : ProcessTestBase
         [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
         public void TestChangesInConsoleEncoding()
         {
```

Comment:
I suggest just changing 
https://github.com/dotnet/runtime/blob/c0b87c70e42c98b03c3148a3a84eb6898ae3eb1c/src/libraries/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs#L46
to
```c#
if (PlatformDetection.IsWindowsServerCore || PlatformDetection.IsWindowsNano)
```

I think your observation in #42000 that the test app is behaving differently to the test is interesting, but for the purposes of getting Server Core going, the above change just recognizes the test is behaving same as Nano.

Reply:
Sounds reasonable. Will update to do that.